yarn.lock

```diff
@@ -139,6 +139,27 @@
     "@babel/traverse" "^7.4.4"
     "@babel/types" "^7.4.4"
 
+
```

Comment:
package.json and yarn.lock are conflicting with master. You'll need to merge master into this branch, and resolve the merge conflict by copying the files from master. (But don't forget to keep the fix for the `django:serve` script in package.json)